test/e2e/__snapshots__/ClientOptions.test.js.snap

```diff
@@ -25,6 +25,9 @@ Array [
 exports[`Client console.log liveReload disabled 1`] = `
 Array [
   "Hey.",
+  "Failed to load resource: the server responded with a status of 404 (Not Found)",
+  "[WDS] Disconnected!",
```

Comment:
should I add a check to disable `"[WDS] Disconnected!"` message when `hot` &  `liveReload` are disabled ? @Loonride @evilebottnawi

Reply:
I think yes, better output message like `[WDS] hot and live reload was disables`